tests/pfc/test_pfc_pause_lossless.py

```diff
@@ -0,0 +1,350 @@
+        bg_dscp_list (list of integers): DSCP values of background traffic.
+        exp_dur (integer): experiment duration in second.
+        start_delay (float): approximated initial delay to start the traffic.
+        test_traffic_pause_expected (bool): Do you expect test traffic to
+            be stopped? If yes, this should be true; false otherwise.
+        send_pause_frame (bool): True/False depending on whether you want to
+           send pause frame Rx port or not.
+
+    Returns:
+        This function returns nothing.
+    """
+
+    # Disable DUT's PFC watchdog.
+    dut.shell('sudo pfcwd stop')
+
+    vlan_subnet = get_vlan_subnet(dut)
+    pytest_assert(vlan_subnet is not None,
+                  "Fail to get Vlan subnet information")
+
+    gw_addr = vlan_subnet.split('/')[0]
+    # One for sender and the other one for receiver.
+    vlan_ip_addrs = get_addrs_in_subnet(vlan_subnet, 2)
+
+    topo_receiver = create_topology(session=session,
+                                    name="Receiver",
+                                    ports=list(rx_port),
+                                    ip_start=vlan_ip_addrs[0],
+                                    ip_incr_step='0.0.0.1',
+                                    gw_start=gw_addr,
+                                    gw_incr_step='0.0.0.0')
+
+    topo_sender = create_topology(session=session,
+                                  name="Sender",
+                                  ports=list(tx_port),
+                                  ip_start=vlan_ip_addrs[1],
+                                  ip_incr_step='0.0.0.1',
+                                  gw_start=gw_addr,
+                                  gw_incr_step='0.0.0.0')
+    start_protocols(session)
+
+    test_traffic = create_ipv4_traffic(session=session,
+                                       name='Test Data Traffic',
+                                       source=topo_sender,
+                                       destination=topo_receiver,
+                                       pkt_size=DATA_PKT_SIZE,
+                                       duration=exp_dur,
+                                       rate_percent=RATE_PERCENTAGE,
+                                       start_delay=start_delay,
+                                       dscp_list=test_dscp_list,
+                                       lossless_prio_list=test_prio_list)
+
+    bg_priority_list = [b for b in range(8) if b not in test_prio_list]
+    background_traffic = create_ipv4_traffic(session=session,
+                                             name='Background Data Traffic',
+                                             source=topo_sender,
+                                             destination=topo_receiver,
+                                             pkt_size=DATA_PKT_SIZE,
+                                             duration=exp_dur,
+                                             rate_percent=RATE_PERCENTAGE,
```

Comment:
RATE_PERCENTAGE -> 100 - RATE_PERCENTAGE

Reply:
Implemented.
    # Assumption: Line rate percentage of background data traffic
    # is equal to Line rate percentage of test data traffic.
    pytest_assert(2 * RATE_PERCENTAGE <= 100,
        "Value of RATE_PERCENTAGE should not be more than 50!")